tests/parser/features/test_internal_call.py

```diff
@@ -1,16 +1,9 @@
-from decimal import (
-    Decimal,
-)
+from decimal import Decimal
 
 import pytest
 
-from vyper.compiler import (
-    compile_code,
-)
-from vyper.exceptions import (
-    StructureException,
-    TypeMismatch,
-)
+from vyper.compiler import compile_code
+from vyper.exceptions import StructureException, TypeMismatch
```

Comment:
This is what `isort` wants? I would think 2+ imports would be the other way

Reply:
This is what `black` wants, in order to reduce the size of diffs.